help/destinations/catalog/streaming/pegaCDH.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Pega Customer Decision Hub connection
+description: Use the Pega Customer Decision Hub destination in Adobe Experience Platform to send profile attributes and segment membership data to Pega Customer Decision Hub for next-best-action decisioning.
+---
+
+# Pega Customer Decision Hub connection
+
+
+
+## Overview {#overview}
+
+Send profile attributes and segment membership data to Pega Customer Descision Hub for next-best-action decisioning. 
+
+Adobe Experience Platform Profile Segment membership, when loaded into Pega Customer Decision Hub, can be used as predictors in adaptive models and help deliver the right contextual and behavioral data for next-best-action decisioning purposes.
+
+## Use cases 
+
+To help you better understand how and when you should use the Customer Decision Hub destination, here are sample use cases that Adobe Experience Platform customers can solve by using this destination.
+
+### Use case #1
+
+*In Telco:*
+
+*A marketer wants to leverage insights from Data Science model-based next best action as delivered by Pega Customer Decision Hub for customer engagement, Pega Customer Decision Hub is heavily reliant on customer intent – for example "Interested_In_5G", "Interested_in_Unlimited_Dataplan" or "Interest_in_iPhone_accessories".*
+
+### Use case #2
+
+*In Financial Services:*
+
+*A marketer wants to optimize the offers for customers who subscribed or unsubscribed for Pension Plan or Retirement Plan newsletters. The financial services can ingest multiple CustomerID from their own CRM to Adobe Experience Platform, build segments from their own offline data, and send profiles that are entering and exiting the segments to Pega Customer Decision Hub for next-best-action (NBA) decisioning in outbound channels.*
+
+## Prerequisites {#prerequisites}
+
+Before you use Pega Customer Decision Hub destination to export data out of Adobe Experience Platform:
+
+* Configure the Adobe Segment Membership Component in your Pega Customer Decision Hub instance.
```

Comment:
as a Pega customer, would I know what this instruction means or do you think that we need to provide a bit more detail? Something like: 

Configure the Adobe Segment Membership Component in your Pega Customer Decision Hub instance. You can do this in Pega by `[...]`

Reply:
Can I update this section later? We are working on publishing the Component in Pega Marketplace